.github/workflows/continuous-integration.yml

```diff
@@ -38,7 +38,16 @@ jobs:
       uses: ./.github/actions/build_ci
       with:
         target: freertos
-    - name: build for Zephyr
+
+  # Break the zephyr builds into their own job as the common runner was
+  # running out of space when runs were together
+  # Also, as the longest running jobs, this allows them to run in ||
+  zephyr_build_known_good:
```

Comment:
what about stable instead of "good" ?

Reply:
Ack.

Reply:
I really don't like stable.  The know good version does not need to be a release, just a recent point that is know to work.  If it is a release, it does not need to be the most recent stable release.

However "know good version" is a term I have used for decades and I am not the only one:
https://github.com/GoogleChromeLabs/chrome-for-testing?tab=readme-ov-file#json-api-endpoints

I can add _version.  That makes it easier to understand.